src/Url.php

```diff
@@ -58,6 +58,7 @@ public function unsignedPath(): string
         $enlarge = (string)(int)$this->enlarge;
         $encodedUrl = rtrim(strtr(base64_encode($this->imageUrl), '+/', '-_'), '=');
         $ext = $this->extension ?: $this->resolveExtension();
+        $ext = $ext ? mb_strtolower($ext) : null;
```

Comment:
Why `mb_strtolower()`? SImple `strtolower()` will do: imgproxy only supports a limited set of extensions and all of them use latin letters.

Reply:
Also, there's no test

Reply:
Will fix in a day